jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -265,14 +265,14 @@ private void analyzeFile(String pkgName, String inPath, boolean includeNonPublic
                           "Inferred Nonnull param for method: " + sign + " = " + result.toString());
                     }
                   }
-                } catch (Exception e) {
-                  LOG(
-                      DEBUG,
-                      "DEBUG",
-                      "Exception while scanning bytecodes for " + mtd + " " + e.getMessage());
                 }
+                analyzeReturnValue(options, cache, mtd, analysisDriver, sign);
```

Comment:
Ran into the following exception here:

```
Exception in thread "main" com.ibm.wala.util.WalaRuntimeException: bad method bytecodes
	at com.ibm.wala.classLoader.ShrikeIRFactory.makeIR(ShrikeIRFactory.java:53)
	at com.ibm.wala.ssa.DefaultIRFactory.makeIR(DefaultIRFactory.java:70)
	at com.uber.nullaway.jarinfer.DefinitelyDerefedParamsDriver.getAnalysisDriver(DefinitelyDerefedParamsDriver.java:109)
	at com.uber.nullaway.jarinfer.DefinitelyDerefedParamsDriver.analyzeReturnValue(DefinitelyDerefedParamsDriver.java:302)
	at com.uber.nullaway.jarinfer.DefinitelyDerefedParamsDriver.analyzeFile(DefinitelyDerefedParamsDriver.java:275)
	at com.uber.nullaway.jarinfer.DefinitelyDerefedParamsDriver.run(DefinitelyDerefedParamsDriver.java:183)
	at com.uber.nullaway.jarinfer.JarInfer.main(JarInfer.java:106)
Caused by: com.ibm.wala.shrikeCT.InvalidClassFileException: Class file invalid at -1: non-abstract method < Application, Landroid/accessibilityservice/AccessibilityGestureEvent, getMotionEvents()Ljava/util/List; > has no bytecodes
	at com.ibm.wala.classLoader.ShrikeBTMethod.processBytecodesWithShrikeBT(ShrikeBTMethod.java:420)
	at com.ibm.wala.classLoader.ShrikeBTMethod.computeBCInfo(ShrikeBTMethod.java:223)
	at com.ibm.wala.classLoader.ShrikeBTMethod.getBCInfo(ShrikeBTMethod.java:129)
	at com.ibm.wala.classLoader.ShrikeBTMethod.getInstructions(ShrikeBTMethod.java:670)
	at com.ibm.wala.classLoader.ShrikeCTMethod.getInstructions(ShrikeCTMethod.java:45)
	at com.ibm.wala.classLoader.ShrikeIRFactory.makeIR(ShrikeIRFactory.java:51)
```

The underlying cause were those `classes-header.jar` files we are now filtering in `android-jar.py`. However, for consistency, we should probably handle these errors the same as we do those that happen on the `getAnalysisDriver()` call at 256 above.